Content.Server/Pointing/EntitySystems/PointingSystem.cs

```diff
@@ -46,6 +46,8 @@ internal sealed class PointingSystem : EntitySystem
 
         private const float PointingRange = 15f;
 
+        private MapCoordinates playerCoordinates;
```

Comment:
this being on the serverside system means you can't have more than one person pointing at the same time